docs/en/core-services/Caching.md

```diff
@@ -223,6 +219,17 @@ available options are:
 - `alluxio.worker.block.annotator.LRUAnnotator`
 - `alluxio.worker.block.annotator.LRFUAnnotator`
 
+#### Evictor Emulation
+The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
+Alluxio supports emulation mode which annotates blocks based on custom evictor implementation. The emulation assumes the configured eviction policy creates
+an eviction plan based on some kind of order and works by regularly extracting this order to be used in annotation activities.
+
+The old evictor configurations should be changes as below. (Failing to change the lef-over configuration will cause class load exceptions as old evictor implementations are deleted.)
```

Comment:
Could you add a sample error message? One common way users find the docs is by searching the exception they see. You could also add it to the troubleshooting page if you think its too low level for this page.